core/src/main/scala/akka/persistence/r2dbc/internal/PayloadCodec.scala

```diff
@@ -11,6 +11,8 @@ import io.r2dbc.postgresql.codec.Json
 import io.r2dbc.spi.Row
 import io.r2dbc.spi.Statement
 
+// move this file into internal/codec dir?
+
```

Comment:
Sounds reasonable, the JSON one as well in that case

Reply:
> the JSON one as well in that case

Moved the whole file, so assuming this is what you meant?!